runtime/compiler/z/env/J9CPU.cpp

```diff
@@ -415,215 +219,78 @@ CPU::getS390SupportsGuardedStorageFacility()
 ////////////////////////////////////////////////////////////////////////////////
 
 void
-CPU::initializeS390zLinuxProcessorFeatures()
+CPU::initializeS390ProcessorFeatures()
```

Comment:
`initializeS390zLinuxProcessorFeatures` and `initializeS390zOSProcessorFeatures` are now unified into this function. I suggest to reviewers to compare the before and after for both functions against this new one and convince themselves that they are indeed the same semantically.